### PR TITLE
README Update for new docs.pupil-labs.com

### PR DESCRIPTION
**Pull request by willpatera**
Make copy text more friendly to general users as well as developers
Move placeholder image to top of document
TODO - add final image before PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,54 @@
...
+- [Develop a Plugin](): Plugins are loaded at runtime from the app bundle. You don't need to setup dependencies and run from source unless you really need bare metal access.
+- [Run from Source](#installing-dependencies): Can't do what you need to do with the network based api or plugin? Then get ready to dive into the inner workings of Pupil and run from source!
 
 ### Installing Dependencies
+* [Ubuntu 18.04 LTS](./docs/dependencies-ubuntu18.md "Pupil dependency installation for Ubuntu 18.04") (recommended Linux distribution)
+* [Ubuntu 17.10 or lower](./docs/dependencies-ubuntu17.md "Pupil dependency installation for Ubuntu 17.10 or lower")
+* [macOS](./docs/dependencies-macos.md "Pupil dependency installation for macOS")
+* [Windows 10](./docs/dependencies-windows.md "Pupil dependency installation for Windows 10")
+
+### Running from source
```
Comment (willpatera) [inline README.md]: Does this section feel out of place here?

Comment (pfaion) [reply README.md]: This is now outdated... take a look at the latest version on my branch.
What do you mean by out of place?
That it should be at a different place in the README or that it should not be in the README at all?

File: README.md [hunk 2]
```diff
@@ -1,28 +1,54 @@
...
-## Getting Started
+## Documentation
+Read the user guide and developer docs at [docs.pupil-labs.com](https://docs.pupil-labs.com "Pupil Labs user guide and develper docs"). 
 
-<p align="center"><img src="https://via.placeholder.com/640x320?text=PLACEHOLDER"/></p>
+## Users
+[Download the latest apps](https://github.com/pupil-labs/pupil/releases/latest "Download Pupil Capture, Pupil Player, and Pupil Service application bundles")! You don't need to know how to write code to _use_ Pupil.
 
-## Building and Running Pupil from Source
+## Develelopers
```
Comment (pfaion) [inline README.md]:
```suggestion
## Developers
```

**Comment by willpatera** [comment]
@pfaion I have resolved merge conflicts - this should be good to go for final PR to pupil-labs/pupil repo.

When you make this PR to pupil-labs/pupil - please make a note that this PR is **waiting on** the new docs.pupil-labs.com to go online.

**Comment by willpatera** [comment]
Fixed a few more errors, now ready to :rocket:

**Comment by pfaion** [comment]
Hi @willpatera 
This PR can be closed, right? Everything has been merged into master as I can see?

**Comment by willpatera** [comment]
Yes, closing as it was merged --> master.

**Pull request closed without merging**